### PR TITLE
Section 4

### PR DESCRIPTION
**Pull request by chicco785**

### CODE REVIEW

File: docs/4.operation/2.deep-log.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Deep Log Inspection
+
+Deep Log Inspection, if you completed the registration of your logs in
+the FIWARE Lab service, allows you to analyse the logs. The service is
+not yet in production, but it will be available soon at:
+<https://kibana.deeplogmanager.lab.fiware.org>.
```
Comment (flopezag) [inline docs/4.operation/2.deep-log.md]: It should be something like:

`[*https://kibana.deeplogmanager.lab.fiware.org*](https://kibana.deeplogmanager.lab.fiware.org)`

Nevertheless the site apparently it is not working at the moment...

File: docs/4.operation/3.fi-health.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## FI-Health
+
+FI-Health is an end-to-end testing tool that ensures that the OpenStack
+integrated components is working as expected. Each of the FIWARE Lab
```
Comment (flopezag) [inline docs/4.operation/3.fi-health.md]: integrated components are ...

File: docs/4.operation/3.fi-health.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+node is tested in a real-world scenario using the OpenStack Rest API
```
Comment (flopezag) [inline docs/4.operation/3.fi-health.md]: OpenStack Rest APIs

File: docs/4.operation/3.fi-health.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+provided by the federated Keystone service endpoints. The idea behind
+these tests is to provide end-to-end testing of the FIWARE Cloud Portal
+to test whether the flow of the application is performing as designed
+from start to finish. The purpose of carrying out these end-to-end tests
```
Comment (flopezag) [inline docs/4.operation/3.fi-health.md]: from start to end

File: docs/4.operation/3.fi-health.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+is to identify OpenStack services problems before or at the same time
+that the users can observe those problems in the FIWARE Cloud Portal.
+Therefore, FI-Health is the key component to provide quality of service
```
Comment (flopezag) [inline docs/4.operation/3.fi-health.md]: Quality of Service

File: docs/4.operation/3.fi-health.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+in the FIWARE Cloud environment.
```
Comment (flopezag) [inline docs/4.operation/3.fi-health.md]: in the FIWARE Lab cloud environment.

File: docs/4.operation/3.fi-health.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+![Sanity Check tool](image6.png)
+
+You can obtain more information about the component by accessing the
+[*FIWARE Health GitHub
+repository*](https://github.com/telefonicaid/fiware-health/blob/develop/README.rst).
+Note that if you want to increase or improve this tool you can suggest
+new issues in this GitHub repository.
```
Comment (flopezag) [inline docs/4.operation/3.fi-health.md]: new issues in this GitHub repository or even better provide us new PRs to extend the functionality of the service.

File: docs/4.operation/4.infographics.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+-   Monitor current status of infrastructure services and know about any
+    issue in any node of FIWARE Lab infrastructure.
+
+While the information on infrastructure capacities is more related to
+marketing, the one on services status is extremely important to support
+Developers and Federation Managers operations. The production URL of the
+Infographics is the following:
+
+-   <http://infographic.lab.fiware.org/>
```
Comment (flopezag) [inline docs/4.operation/4.infographics.md]: `[*http://infographic.lab.fiware.org*](http://infographic.lab.fiware.org)`

File: docs/4.operation/4.infographics.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+-   <http://status.lab.fi-ware.org/>
```
Comment (flopezag) [inline docs/4.operation/4.infographics.md]: `[*http://status.lab.fi-ware.org*](http://status.lab.fi-ware.org)`

File: docs/4.operation/4.infographics.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+This component provides capacity information about the different
+OpenStack regions and status information of infrastructure services. The
+Federation Monitor component exposes data on the capacity and status of
+FIWARE Lab infrastructure through a set of RESTful API that Infographics
+and Status Pages calls in order to get that data.
```
Comment (flopezag) [inline docs/4.operation/4.infographics.md]: call

File: docs/4.operation/4.infographics.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+For more details, it is recommended to refer to the main documentation
+included in the software repository:
+<https://github.com/SmartInfrastructures/fi-lab-infographics>.
```
Comment (flopezag) [inline docs/4.operation/4.infographics.md]: `[*https://github.com/SmartInfrastructures/fi-lab-infographics*](https://github.com/SmartInfrastructures/fi-lab-infographics)`

File: docs/4.operation/4.infographics.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+Figure below shows the main page of the Infographics tool. It is comprised
+of:
+
+-   a user’s section: it shows the total number of users as sum of Basic
```
Comment (flopezag) [inline docs/4.operation/4.infographics.md]: ... as sum of Trial and Community users;

File: docs/4.operation/4.infographics.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+    Users, Trial Users and Community Users;
+
+-   a map that shows all nodes;
+
+-   six interactive tabs that display different data;
```
Comment (flopezag) [inline docs/4.operation/4.infographics.md]: display different data (number of regions, total number of cores, RAM, and HDD, total number of IPs and the total number of virtual machines currently deployed);

File: docs/4.operation/4.infographics.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+
+![FIWARE Infographics main page](image7.png)
+
+The Status page depicted below is composed by:
+
+-   a big map that shows all the nodes and their overall status (green,
+    yellow or red);
+
+-   a table showing the current status of all the FIWARE Lab nodes
+    services together with the FIHealth Sanity Check status (FIHealth
```
Comment (flopezag) [inline docs/4.operation/4.infographics.md]: FI-Health

File: docs/4.operation/5.rancher.md [hunk 15]
```diff
@@ -0,0 +1,22 @@
+## Rancher
+
+FIWARE Ops team is currently deploying a customised version of
```
Comment (flopezag) [inline docs/4.operation/5.rancher.md]: Check this sentence, is currently deploying or has deployed... (and where?)

File: docs/4.operation/5.rancher.md [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+[Rancher 1.6](https://rancher.com/docs/rancher/v1.6/en/) on top of FIWARE Lab.
```
Comment (flopezag) [inline docs/4.operation/5.rancher.md]: `[Rancher 1.6](https://rancher.com/docs/rancher/v1.6/en)`

File: docs/4.operation/5.rancher.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+-   Their OpenStack APIs are reachable from FIWARE Lab servers (which
+    should be the default case).
+
+-   Provide the configuration of MTU within their node to users
+    interested into using the service (or even installing manually
+    Docker Swarm).
+
+For more details, we recommend you to refer to the documentation of the
+Rancher set-up within FIWARE Lab:
+[*https://github.com/SmartInfrastructures/rancher-setup-notes*](https://github.com/SmartInfrastructures/rancher-setup-notes)
```
Comment (flopezag) [inline docs/4.operation/5.rancher.md]: page not found...

File: docs/4.operation/5.rancher.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+This tool allows users with a Cloud account in FIWARE Lab to deploy and
+manage a Docker Swarm cluster on FIWARE Lab. Through which they can
+deploy their FIWARE applications on the Lab using docker containers.
+This is a service offered to the users, and as such FIWARE Lab Node
+Administrators do not have much to do. They only need to be sure that:
+
+-   Their OpenStack APIs are reachable from FIWARE Lab servers (which
+    should be the default case).
+
+-   Provide the configuration of MTU within their node to users
```
Comment (flopezag) [inline docs/4.operation/5.rancher.md]: which MTU? and how to do it?

File: docs/4.operation/5.rancher.md [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+    interested into using the service (or even installing manually
+    Docker Swarm).
+
+For more details, we recommend you to refer to the documentation of the
+Rancher set-up within FIWARE Lab:
+[*https://github.com/SmartInfrastructures/rancher-setup-notes*](https://github.com/SmartInfrastructures/rancher-setup-notes)
+
+![FIWARE Rancher tool.](image9.png)
```
Comment (flopezag) [inline docs/4.operation/5.rancher.md]: `FIWARE Rancher tool`

File: docs/4.operation/6.mastermind.md [hunk 20]
```diff
@@ -0,0 +1,13 @@
+## FIWARE Mastermind
+
+MasterMind is an ongoing development effort to provide a solution to
+easily configure and orchestrate FIWARE NGSI-based services. The service
+is provided to FIWARE Lab end-users as a centralised service. No
+specific action is needed by single FIWARE Lab Node Administrators.
+Currently the service in under development, so it is not yet deployed in
```
Comment (flopezag) [inline docs/4.operation/6.mastermind.md]: Check this sentence, probably it is obsolete...

"Currently the service in under development..."

File: docs/4.operation/7.jira.md [hunk 21]
```diff
@@ -0,0 +1,14 @@
+## JIRA
+
+Jira is the ticket manager tool used from internal and external users.
+
+The main internal usage of Jira is to keep track of all activities
+carried out within FIWARE Lab and to assign specific tasks to the
+different stakeholders. Jira, combined to Agile dynamics is used to
```
Comment (flopezag) [inline docs/4.operation/7.jira.md]: Jira, combined to Agile dynamics, is used to

File: docs/4.operation/7.jira.md [hunk 22]
```diff
@@ -0,0 +1,14 @@
...
+create monthly Sprint useful to plan upcoming tasks as well as to fix
+and reach targets.
+
+The following is a picture of Jira which is available at
+<https://jira.fiware.org/>
```
Comment (flopezag) [inline docs/4.operation/7.jira.md]: `[https://jira.fiware.org](https://jira.fiware.org)`

File: docs/4.operation/8.backlog.md [hunk 23]
```diff
@@ -0,0 +1,41 @@
+## Backlog
```
Comment (flopezag) [inline docs/4.operation/8.backlog.md]: In general, we want to dismiss this tool by the end of FI-Next project. Therefore, I recommend that you do not include the description of it inside this repository...

File: docs/index.md [hunk 24]
```diff
@@ -25,3 +25,11 @@ and connect it to the FIWARE Lab.
...
     - [Registering your Node in the Infographics](3.process/9.infographics.md)
     - [Registering your Node in Deep Log Inspection](3.process/10.log.md)
+1. [Operating and Maintaining a FIWARE Lab node](4.operation/1.introduction.md)
+    - [Deep Log Inspection](4.operation/2.deep-log.md)
+    - [FI-Health](4.operation/3.fi-health.md)
+    - [Infographics](4.operation/4.infographics.md)
+    - [Rancher](4.operation/5.rancher.md)
+    - [Mastermind](4.operation/6.mastermind.md)
+    - [JIRA](4.operation/7.jira.md)
+    - [Backlog](4.operation/8.backlog.md)
```
Comment (flopezag) [inline docs/index.md]: Delete this link:

`- [Backlog](4.operation/8.backlog.md)`

File: mkdocs.yml [hunk 25]
```diff
@@ -27,6 +27,14 @@ pages:
...
       - 'Registering your Node in the Infographics': '3.process/9.infographics.md'
       - 'Registering your Node in Deep Log Inspection': '3.process/10.log.md'
+    - 'Operating and Maintaining a FIWARE Lab node':
+      - 'Introduction': '4.operation/1.introduction.md'
+      - 'Deep Log Inspection': '4.operation/2.deep-log.md'
+      - 'FI-Health': '4.operation/3.fi-health.md'
+      - 'Rancher': '4.operation/5.rancher.md'
+      - 'Mastermind': '4.operation/6.mastermind.md'
+      - 'JIRA': '4.operation/7.jira.md'
+      - 'Backlog': '4.operation/8.backlog.md'
```
Comment (flopezag) [inline mkdocs.yml]: Delete this link:

`- [Backlog](4.operation/8.backlog.md)`

File: docs/4.operation/8.backlog.md [hunk 26]
```diff
@@ -0,0 +1,41 @@
...
+
+The backlog management web site is a tool implemented with the purpose
+to provide easy understanding of the activity recorded on the trackers.
+Access to FIWARE Backlog Management Web Site at
+[*http://backlog.fiware.org*](http://backlog.fiware.org)
+
+![FIWARE Backlog tool.](image12.png)
```
Comment (flopezag) [inline docs/4.operation/8.backlog.md]: Please, delete this image from the repository as well.

File: docs/4.operation/8.backlog.md [hunk 27]
```diff
@@ -0,0 +1,41 @@
...
+
+At FIWARE we are looking to be sensitive to several sources demanding
+both reactive and proactive working modes. Therefore, the Backlog
+Management web site has been arranged accordingly.
+
+![FIWARE Backlog, urgent desk management](image13.png)
```
Comment (flopezag) [inline docs/4.operation/8.backlog.md]: Please, delete this image from the repository as well.

File: docs/4.operation/8.backlog.md [hunk 28]
```diff
@@ -0,0 +1,41 @@
...
+option in the menu. The Urgent Desk is an important tool since it allows
+to create common awareness on issues with time-sensitive fields such as
+issues’ deadlines, priorities and status. To meet this objective, it
+collects all items in the trackers with relevant deadlines, priorities
+and status to display them in the desk. Additionally, the Help Desk is
+also a key tool in supporting the end users. It is implemented by adding
+email lists connected to JIRA. The Backlog Web Site also helps by
+providing meaningful views for the different channels and actors.
+
+![FIWARE Backlog, Help-Desk evolution.](image14.png)
```
Comment (flopezag) [inline docs/4.operation/8.backlog.md]: Please, delete this image from the repository as well.

File: docs/4.operation/5.rancher.md (hunk 15 above)
Comment (Cerfoglg) [reply docs/4.operation/5.rancher.md]: @flopezag I recall Rancher was deployed at some point, but I haven't heard anything about it in a while, so I'm not certain of the status on this.

File: docs/4.operation/5.rancher.md (hunk 18 above)
Comment (Cerfoglg) [reply docs/4.operation/5.rancher.md]: @flopezag The MTU for the nodes Rancher runs on I believe.

File: docs/4.operation/5.rancher.md (hunk 17 above)
Comment (flopezag) [reply docs/4.operation/5.rancher.md]: Why did you delete this section? if there is a content about Rancher it makes sense to keep this paragraph to understand how to get details about use of Rancher in FIWARE Lab.

File: docs/4.operation/5.rancher.md (hunk 18 above)
Comment (flopezag) [reply docs/4.operation/5.rancher.md]: The question here is to resolve the ambiguity. If we mention that we should configure the MTU inside the nodes, we should describe in short term how we should it and even more important which value we should assign to the MTU.

File: docs/4.operation/5.rancher.md (hunk 15 above)
Comment (flopezag) [reply docs/4.operation/5.rancher.md]: In a sentence bellow, I think that it was written that it is still not deployed. Nevertheless, it is a question to certify the status of the deployment of Rancher inside the FIWARE Lab.

**Review by flopezag — COMMENTED** [review]
Reviewed the complete comments and added comments to the pending ones.

File: docs/4.operation/5.rancher.md (hunk 17 above)
Comment (Cerfoglg) [reply docs/4.operation/5.rancher.md]: That repository doesn't seem to exist anymore (or it was made private) and I couldn't find if it was moved somewhere, so I took the link out. We could open an issue about this if that resource isn't available right now.

File: docs/4.operation/5.rancher.md (hunk 15 above)
Comment (Cerfoglg) [reply docs/4.operation/5.rancher.md]: Confirmed that Rancher is up, updated with link

File: docs/4.operation/5.rancher.md (hunk 18 above)
Comment (Cerfoglg) [reply docs/4.operation/5.rancher.md]: I added some elaboration

File: docs/4.operation/5.rancher.md (hunk 15 above)
Comment (flopezag) [reply docs/4.operation/5.rancher.md]: LGTM

File: docs/4.operation/5.rancher.md (hunk 17 above)
Comment (flopezag) [reply docs/4.operation/5.rancher.md]: Ok, open an issue about this issue.

File: docs/4.operation/5.rancher.md (hunk 18 above)
Comment (flopezag) [reply docs/4.operation/5.rancher.md]: LGTM

**Review by flopezag — COMMENTED** [review]
Open an issue regarding the problem with the link.

File: docs/4.operation/5.rancher.md (hunk 17 above)
Comment (Cerfoglg) [reply docs/4.operation/5.rancher.md]: Done, issue #15

Comment (flopezag) [reply docs/4.operation/5.rancher.md]: LGTM

**Review by flopezag — APPROVED** [review]
LGTM

**Pull request merged**